Zype/Managers/ACActionSheetManager.m

```diff
@@ -587,7 +587,21 @@ + (UIActionSheet *)videoDetailPlayAsActionSheet{
     
     UIActionSheet *actionSheet = [[UIActionSheet alloc] initWithTitle:nil delegate:nil cancelButtonTitle:cancelButtonTitle destructiveButtonTitle:nil otherButtonTitles:nil];
     
-    [actionSheet addButtonWithTitle:[self titleForPlayAsActionSheetButtonWithType:ACActionSheetPlayAsButtonListen]];
+    BOOL isVideo = NO;
+    BOOL isAudio = NO;
```

Comment:
@advantiss425 - this seems to be too restrictive.

Download as audio is limited to m4a

Play as audio supports m4a, mp3, or hls audio renditions.